test/viewmodels/ImportVatViewModelSpec.scala

```diff
@@ -0,0 +1,384 @@
+        shouldContainCorrectLast6MonthsHeading(viewModelWithCurrentCerts)
+        shouldNotContainNotificationPanel(viewModelWithCurrentCerts)
+        shouldContainCurrentStatements(viewModelWithCurrentCerts, certificatesForAllEoris)
+        shouldNotContainCurrentStatementsNotAvailableGuidance(viewModelWithCurrentCerts)
+        shouldContainCorrectCertsOlderThan6MonthsGuidance(viewModelWithCurrentCerts)
+        shouldContainCorrectChiefDeclarationGuidance(viewModelWithCurrentCerts)
+        shouldContainCorrectHelpAndSupportGuidance(viewModelWithCurrentCerts)
+      }
+
+      "model has both requested and current certificates" in new Setup {
+        shouldContainCorrectTitle(viewModel)
+        shouldContainCorrectBackLink(viewModel)
+        shouldContainCorrectHeading(viewModel)
+        shouldContainCorrectCertificateAvailableGuidance(viewModel)
+        shouldContainCorrectLast6MonthsHeading(viewModel)
+        shouldContainNotificationPanel(viewModel)
+        shouldContainCurrentStatements(viewModel, certificatesForAllEoris)
+        shouldNotContainCurrentStatementsNotAvailableGuidance(viewModel)
+        shouldContainCorrectCertsOlderThan6MonthsGuidance(viewModel)
+        shouldContainCorrectChiefDeclarationGuidance(viewModel)
+        shouldContainCorrectHelpAndSupportGuidance(viewModel)
+      }
+
+      "model has neither requested nor current certificates" in new Setup {
+        shouldContainCorrectTitle(viewModelWithNoCerts)
+        shouldContainCorrectBackLink(viewModelWithNoCerts)
+        shouldContainCorrectHeading(viewModelWithNoCerts)
+        shouldContainCorrectCertificateAvailableGuidance(viewModelWithNoCerts)
+        shouldContainCorrectLast6MonthsHeading(viewModelWithNoCerts)
+        shouldNotContainNotificationPanel(viewModelWithNoCerts)
+        shouldNotContainCurrentStatements(viewModelWithNoCerts)
+        shouldContainCurrentStatementsNotAvailableGuidance(viewModelWithNoCerts)
+        shouldContainCorrectCertsOlderThan6MonthsGuidance(viewModelWithNoCerts)
+        shouldContainCorrectChiefDeclarationGuidance(viewModelWithNoCerts)
+        shouldContainCorrectHelpAndSupportGuidance(viewModelWithNoCerts)
+      }
+    }
+  }
+
+  private def shouldContainCorrectTitle(viewModel: ImportVatViewModel)(implicit msgs: Messages): Assertion =
+    viewModel.title mustBe Some(msgs("cf.account.vat.title"))
+
+  private def shouldContainCorrectBackLink(viewModel: ImportVatViewModel)(implicit config: AppConfig): Assertion =
+    viewModel.backLink mustBe Some(config.customsFinancialsFrontendHomepage)
+
+  private def shouldContainCorrectHeading(viewModel: ImportVatViewModel)(implicit msgs: Messages): Assertion =
+    viewModel.heading mustBe h1Component(
+      "cf.account.vat.title",
+      id = Some("import-vat-certificates-heading"),
+      classes = "govuk-heading-xl"
+    )
+
+  private def shouldContainCorrectCertificateAvailableGuidance(viewModel: ImportVatViewModel)(implicit
```

Comment:
I've made a note about the implicit groupings earlier and it's here as well. I'm not sure it needs to change, as long as we're consistent and think it's fine.